Packs/CommonScripts/Scripts/TextFromHTML/README.md

```diff
@@ -9,26 +10,44 @@ Extract regular text from the given HTML
 | Tags | Utility |
 | Cortex XSOAR Version | 5.0.0 |
 
+## Used In
+
+---
+This script is used in the following playbooks and scripts.
+
+* TextFromHTML_test_playbook
+
```

Comment:
Remove this. Test playbooks are just used internally (like unit-tests) and aren't shipped / used by users.
```suggestion
```

Reply:
Wasn't intentional, generate-docs automatically seems to add this.

Reply:
That's odd, the SDK should skip those. I'll open an internal ticket for fixing that.